tests/CppWinRTComServerTests.cpp

```diff
@@ -20,15 +20,17 @@ void notifier()
     _comExit.SetEvent();
 }
 
-struct MyServer : winrt::implements<MyServer, winrt::Windows::Foundation::IStringable>
+struct __declspec(uuid("8D0E7037-9855-4659-B48D-F5DBD50F7836")) MyServer

```

Comment:
These are meant to be winrt type types so they wouldn't explicitly define UUID. Please revert this and add `MyComClass` for testing support for COM classes instead, thanks!

Reply:
If this is removed, `winrt::guid_of` returns the UUID of `IStringable`, which is definitely not correct behavior for this purpose. `CoCreateClass(__uuidof(ISomeInterface))` should not be valid.

Reply:
> These are meant to be winrt type types so they wouldn't explicitly define UUID.

However, they are being registered to a *COM* server so they need a CLSID. The fact these classes implement WinRT interfaces does not detract from this.